internal/raft/message/append_entries.proto

```diff
@@ -11,7 +11,7 @@ message AppendEntriesRequest {
     int32 prevLogIndex = 3;
     int32 prevLogTerm = 4;
     repeated LogData Entries = 5;
-    int32 leaderCommit = 6;
```

Comment:
Add actual Godoc comments instead of this, so that there is actual godoc in the generated source!